Detections/AWSCloudTrail/AWS_ChangeToRDSDatabase.yaml

```diff
@@ -1,15 +1,17 @@
 ﻿id: 8c2ef238-67a0-497d-b1dd-5c8a0f533e25
 name: Changes to internet facing AWS RDS Database instances
 description: |
-  'Monitoring Changes to internet facing AWS RDS (Relational Database Service) Database instances.
+  'Amazon Relational Database Service (RDS) is scalable relational database in the cloud. 
+  If your organization have AWS RDS Database running, monitoring changes to especially internet facing AWS RDS (Relational Database Service) 
+  Database instances is crucial. Once alerts triggered, validate if changes observed are authorized and adhere to change control policy. 
   More information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255
-  and https://aws.amazon.com/rds/.'
+  and RDS API Reference Docs: https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_Operations.html'
 severity: Low
 requiredDataConnectors:
   - connectorId: AWS
     dataTypes:
       - AWSCloudTrail
-queryFrequency: 1d
+queryFrequency: 1h
```

Comment:
I don't think we wan to do 1h.  1d is generally the lowest we want unless it is a High Severity that we know is a 99% TP.  This does reduce the number of alerts at a given time, but it will potentially give them several alerts an hour versus many alerts a day.